menu/reader.js

```diff
@@ -0,0 +1,42 @@
+const fs = require("fs");
+const YAML = require("js-yaml");
+
+class Menu {
+    constructor(text) {
+        this.delivery = text['Delivery'];
+        this.instructions = text['Instructions']; // If instructions provided
+        this.dishes = text['Menu'];
+        this.modifiers = text['Modifiers'];
+    }
+
+}
+
+// Convert .yml file to yml text
+function readYmlFile(filename) {
+    try {
+        const raw = fs.readFileSync(filename, 'utf-8');
+        return raw
+    } catch (e) {
+        throw new Error("File cannot be Read!");
```

Comment:
Better to print this message then throw the same error upward. By throwing a new error, we would be discarding potentially information that the caught error contains.

Reply:
Agreed on this. I'll print the error messages for both cases as I think it's applicable to the yml parsing as well